debug_type_filtering.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
These debug.sh-files quite regularly contain usernames and passwords, that is quite tricky